moveit_setup_assistant/templates/moveit_config_pkg_template/launch/gazebo.launch

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="paused" default="false"/>
+  <arg name="gui" default="true"/>
+  
+  <!-- startup simulated world -->
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+    <arg name="world_name" default="worlds/empty.world"/>
+    <arg name="paused" value="$(arg paused)"/>
+    <arg name="gui" value="$(arg gui)"/>
+  </include>
+
+  <!-- send robot urdf to param server -->
+  <param name="robot_description" [URDF_LOAD_ATTRIBUTE] />
+
+  <!-- push robot_description to factory and spawn robot in gazebo -->
+  <node name="spawn_gazebo_model" pkg="gazebo_ros" type="spawn_model" args="-urdf -param robot_description -model robot -z 0.1" respawn="false" output="screen" />
```

Comment:
do you understand why ``-z 0.1`` is used?

i believe this may be a custom value for a particular robot that is not very generic. please add a comment in this launch file explaining the purpose of ``z`` (changing the start location of the robot). i think it should default to 0

Reply:
Yes that is right, 0 as default makes better sense, thanks.

Reply:
Changed it, now robot spawn at the origin.

Reply:
Actually can you leave the ``-z`` option there but set it to zero and document its usage?

Reply:
If we leave the `z`, shouldnt we also leave the -x and -y?

Reply:
great idea!